datasource/src/main/resources/application.conf

```diff
@@ -14,4 +14,21 @@ database {
   liquibase {
     changeLogResourcePath = "liquibase/changelog/changelog-master.xml"
   }
+
+  mongo {
```

Comment:
I wonder why we need it here. We already have configurations for mongo in `portal` module, so why duplicate them here? I assume there is a reason for it, so I'm just asking

Reply:
We need it in integration tests, somehow it takes config from here